.github/prlint.json

```diff
@@ -0,0 +1,32 @@
+{
+    "title": [
+        {
+            "pattern": "^(docs|feat|fix|dev|test):\\s",
```

Comment:
Let's add 'ref', 'build' and 'perf' too.

Reply:
Disregard that comment!

I was thinking on simplifying it a bit and make these map to the types of issues on our labels (https://github.com/britecharts/britecharts/labels)

So, that would become:
fix - for bug fixes
feat - for features
docs - for documentation stuff
main - for all the rest (refactors, build, tests, dev environment, etc.)

What do you think?